### ISSUE TITLE
Custom Model

### ISSUE THREAD

**Issue by vibhavnirmal** [yahiya6006/Face-Mask-detector-using-tensorflow-lite-model#1]
How to train model on custom datasets. Any resource link would be appreciated.

**Comment by yahiya6006** [comment]
Hey,
You can train your own deep neural network for this purpose or use the TensorFlow object detection API. I have used the API to train this model. You will get many tutorial posts/videos available in Medium, YouTube etc.

**Comment by vibhavnirmal** [comment]
Okay... Can you please share any links you have used.. if it is handy? I will surely check out myself on you tube and Medium. Thank you
